models/DataObject/ClassDefinition/DynamicOptionsProvider/MultiSelectOptionsProviderInterface.php

```diff
@@ -31,4 +34,6 @@ public function getOptions(array $context, Data $fieldDefinition): array;
      *
      */
     public function hasStaticOptions(array $context, Data $fieldDefinition): bool;
+
+    public function getDefaultValue(array $context, Data $fieldDefinition): ?string;
```

Comment:
This is a breaking change. Do we actually need this here?

Reply:
Would it be a better solution if I remove it from the code and instead highlight in the documentation that users can utilize the SelectOptiontProviderInterface to make use of the default value functionality?"